fighthealthinsurance/views.py

```diff
@@ -1,37 +1,47 @@
 import json
-from typing import *
+import stripe
+import typing
 
+from PIL import Image
+from django import forms
 from django.conf import settings
 from django.shortcuts import redirect, render
 from django.urls import reverse
-from django.views import View, generic
 from django.utils.safestring import mark_safe
+from django.views import View, generic
 
-import stripe
-from fighthealthinsurance.common_view_logic import *
-from fighthealthinsurance.core_forms import *
-from fighthealthinsurance.generate_appeal import *
-from fighthealthinsurance.models import *
-from fighthealthinsurance.question_forms import *
-from fighthealthinsurance.utils import *
+from fighthealthinsurance import common_view_logic
+from fighthealthinsurance import core_forms
+from fighthealthinsurance import generate_appeal
+from fighthealthinsurance import models
+from fighthealthinsurance import question_forms
+from fighthealthinsurance import utils
+
+appealGenerator = generate_appeal.AppealGenerator()
 
-appealGenerator = AppealGenerator()
+
+def render_ocr_error(request, text):
+    return render(request, "server_side_ocr_error.html", context={
+        "error": text,
+    })
```

Comment:
Why is it up here?

Reply:
Doesn't reference `self`, so no need to be a method on the class. I had it as a `@staticmethod` instead but I usually lean to just putting local utility functions in the module scope instead of on a class. It doesn't really make much of a difference to me, though.